templates/zerver/help/enable-emoticon-translations.md

```diff
@@ -79,7 +79,7 @@ automatically by Zulip.
             <td align="center"><code>:/</code></td>
             <td align="center">
                 <img
-                    src="/static/generated/emoji/images/emoji/confused.png"
+                    src="/static/generated/emoji/images-google-64/1f615.png"
```

Comment:
We should probably plan to make this page templated from the data at some point, but definitely not a blocker for merging this.